dev/breeze/src/airflow_breeze/utils/packages.py

```diff
@@ -443,7 +443,9 @@ def get_install_requirements(provider_id: str, version_suffix: str) -> str:
         dependencies = get_provider_requirements(provider_id)
     else:
         dependencies = PROVIDER_DEPENDENCIES.get(provider_id)["deps"]
-    install_requires = [apply_version_suffix(clause, version_suffix) for clause in dependencies]
+    install_requires = [
+        apply_version_suffix(clause, version_suffix).replace('"', '\\"') for clause in dependencies
```

Comment:
Does the python version in the dependency actually _need_ to be quoted? As the version is also w/o quotes...

Reply:
Yes

Reply:
https://peps.python.org/pep-0508/#environment-markers